libeos-payg/tests/manager.c

```diff
@@ -204,6 +204,7 @@ test_manager_disabled (Fixture *fixture,
     remove_path (fixture->key_path);
 
   g_autoptr(GError) error = NULL;
+  gint64* time_added = 0;
```

Comment:
```suggestion
  gint64 *time_added = 0;
```

and in every other function where this is added.